### DISCUSSION TITLE
siunitx包：分辨率 \numproduct{1920x1080} 的像素数量约为

### DISCUSSION THREAD

**Discussion by lingr7** [nuaatug/nuaathesis#69] [General]
原文为：分辨率 \num{1920x1080} 的像素数量约为
demo_chs/master-ZS 报错了。
修正为：分辨率 \numproduct{1920x1080} 的像素数量约为
才正常编译输出。
需要一个PR

**Comment by yzwduck** [comment]
感谢，已更新 siunitx 包相关示例 1850628337fd69fe363875a5f4b84d853eda9ab3。

**Reply by lingr7** [reply]
好的，发现demo.tex中关于表格无法正常编译的地方也被修复了。感谢维护者的及时修复。
